src/main/webapp/reducers/projectReducer.js

```diff
@@ -2,30 +2,44 @@ import { ACTION } from '../actions/types';
 
 const initialState = {
   activeProject: '',
-  activeUser: '',
+  html: '',
+  css: '',
+  js: '',
+  title: '',
+  activeCollaborator: '',
```

Comment:
Why do we need both collaborator and collaboratorName? 

Also, activeCollaborator is not clear what type of data it holds. I'd recommend updating the field name to indicate exactly what it is, similar to what you did for the name.